include/hydrogen_charge_exchange.hxx

```diff
@@ -69,7 +69,7 @@ protected:
   ///  atom_energy  Energy removed from atom1, added to ion2
   ///  ion_energy   Energy removed from ion1, added to atom2
   ///
-  void calculate_rates(Options& atom1, Options& ion1, Options& atom2, Options& ion2,
+  void calculate_rates(GuardedOptions atom1, GuardedOptions ion1, GuardedOptions atom2, GuardedOptions ion2,
```

Comment:
Another place where we're passing by value rather than by reference -- is this intentional?